src/components/MovieDetails/MovieDetails.js

```diff
@@ -7,21 +7,21 @@ import { addUserRating, getRatings, deleteRating } from '../../apiCalls';
 
 
 export const MovieDetails = ({ location, user, ratings, setRatings }) => {
-  
+
   const removeRating = async (event) => {
-    let movieID = parseInt(event.target.id);
-    await deleteRating(user.id, movieID)
+    let ratingID = event.target.id;
```

Comment:
Nice job changing the name of that to make it more relevant to what it actually is